.pipelines/pypl-publishing.yml

```diff
@@ -44,6 +49,11 @@ parameters:
   type: string
   default: '1.18.0-dev-20240426-0116-b842effa29'
 
+- name: ort_rocm_version
+  displayName: 'OnnxRuntime ROCm version'
+  type: string
+  default: '1.19.0-dev-20240602-1103-217b66f'
```

Comment:
I found no 1.18 build with ROCm and only 1.19.0 build suits here. I could be wrong.